fsspec/spec.py

```diff
@@ -1013,13 +1016,18 @@ def expand_path(self, path, recursive=False, maxdepth=None):
                     if recursive:
                         out |= set(
                             self.expand_path(
-                                list(bit), recursive=recursive, maxdepth=maxdepth
+                                list(bit),
+                                recursive=recursive,
+                                maxdepth=maxdepth,
+                                **kwargs,
```

Comment:
What about `glob`, a few lines above?

Reply:
Yes, these kwargs should also be passed to `glob`. It's fixed now!